cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+describe('', () => {
+  before(() => {
+    
+  });
+  
+  
+  it('', () => {
+   cy.visit('https://react-redux.realworld.io/#/login');
+   cy.get(':nth-child(1) > .form-control')
+     .type('kakoc.pwn@gmail.com')
+    cy.get(':nth-child(2) > .form-control')
+    .type('321678')
```

Comment:
It's better when chained methods have the same indentation
```suggestion
      .type('321678')
```